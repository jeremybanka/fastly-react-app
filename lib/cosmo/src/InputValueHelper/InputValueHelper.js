/* @flow */

import * as React from "react";
import { FormattedMessage } from "react-intl";
import { Text } from "../";

type TextType =
  | string
  | React.Element<typeof Text>
  | React.Element<typeof FormattedMessage>;

type InputValueHelperProps = {
  htmlFor: string,
  children: TextType,
};
const InputValueHelper = (props: InputValueHelperProps) => {
  return (
    <Text
      as="label"
      htmlFor={props.htmlFor}
      letterSpacing="sm"
      color="grays.4"
      fontSize="xs"
    >
      {props.children}
    </Text>
  );
};

export default InputValueHelper;
