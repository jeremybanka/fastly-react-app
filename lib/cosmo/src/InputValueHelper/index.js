/* @flow */

import InputValueHelper from "./InputValueHelper";

export default InputValueHelper;
