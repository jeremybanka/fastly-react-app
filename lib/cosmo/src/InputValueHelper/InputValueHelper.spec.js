/* @flow */

import React from "react";
import { render } from "enzyme";
import InputValueHelper from "./InputValueHelper";

describe("InputValueHelper test suite", () => {
  const component = <InputValueHelper>Label</InputValueHelper>;
  it("renders the stored snapshot", () => {
    expect(render(component)).toMatchSnapshot();
  });
});
