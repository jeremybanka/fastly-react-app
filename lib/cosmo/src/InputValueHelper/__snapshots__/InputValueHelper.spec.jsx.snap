// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputValueHelper test suite renders the stored snapshot 1`] = `
<label
  class="Text-sc-6c1xd4-0 bYunon"
  color="grays.4"
  font-size="xs"
  letter-spacing="sm"
>
  Label
</label>
`;
